### PR TITLE
Reorg automation docs and add a using blueprints section

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Reorganize automation documentation and add a section about using blueprints.

Note, this is not about creating blueprints.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation.markdown [hunk 1]
```diff
@@ -3,42 +3,12 @@ title: "Automating Home Assistant"
 description: "Steps to help you get automation setup in Home Assistant."
 ---
 
-Home Assistant offers a wide range of automation configurations. In this section, we'll try to guide you through all the different possibilities and options. Besides this documentation, there are also a couple of people who have made their automations [publicly available](/cookbook/#example-configurationyaml).
+Home Assistant contains information about all your devices and services. This information is not only available for the user in the dashboard, it can also be used to trigger automations. And that's fun!
 
-Please see [Automation Integration](/integrations/automation/) for configuration options and services.
+If you are just starting out, we strongly suggest you start with blueprint automations. These are ready made automations by the community that you only need to configure.
```
Comment (frenck) [inline source/_docs/automation.markdown]:
```suggestion
If you are just starting out, we strongly suggest you start with blueprint automations. These are ready-made automations by the community that you only need to configure.
```

File: source/_docs/automation.markdown [hunk 2]
```diff
@@ -3,42 +3,12 @@ title: "Automating Home Assistant"
 description: "Steps to help you get automation setup in Home Assistant."
```
Comment (frenck) [inline source/_docs/automation.markdown:3]: What I mis on this page:

What is automation?!

(we didn't tell that before either thought).

File: source/_docs/automation.markdown (hunk 1 above)
Comment (zsarnett) [inline source/_docs/automation.markdown]:
```suggestion
If you are just starting, we strongly suggest you start with blueprint automations. These are ready-made automations by the community that you only need to configure.
```

File: source/_docs/automation/basics.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Understanding Automations"
+description: "A breakdown of what an automation consists of."
+---
+
+All automations are made up of a trigger and an action. Optionally combined with a condition. Take for example the automation:
+
+> When Paulus arrives home and it is after sunset: Turn the lights on in the living room.".
+
+We can break up this automation in the following three parts:
```
Comment (zsarnett) [inline source/_docs/automation/basics.markdown]:
```suggestion
We can break up this automation into the following three parts:
```

File: source/_docs/automation/basics.markdown [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+| Name | Description | Example |
+| ---- | ----- | ---- |
+| Entity ID | Unique identifier for the entity. | `light.kitchen`
+| State | The current state of the device. | `home`
+| Attributes | Extra data related to the device and/or current state. | `brightness`
+
+State changes can be used as the source of triggers and the current state can be used in conditions.
+
+Actions are all about calling services. To explore the available services open the **Developer Tools** -> **Services**. Services allow to change anything. For example turn on a light, run a script or enable a scene. Each service has a domain and a name. For example the service `light.turn_on` is capable of turning on any light in your system. Services can be passed parameters to for example tell which device to turn on or what color to use.
```
Comment (zsarnett) [inline source/_docs/automation/basics.markdown]:
```suggestion
Actions are all about calling services. To explore the available services open the **Developer Tools** -> **Services**. Services allow changing anything. For example turn on a light, run a script or enable a scene. Each service has a domain and a name. For example the service `light.turn_on` is capable of turning on any light in your system. Services can be passed parameters to for example tell which device to turn on or what color to use.
```

File: source/_docs/automation/basics.markdown [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## Creating automations
+
+Now that you've got a sneak peak of what is possible, it's time to get your feet wet and create your first automation.
```
Comment (zsarnett) [inline source/_docs/automation/basics.markdown]:
```suggestion
Now that you've got a sneak peek of what is possible, it's time to get your feet wet and create your first automation.
```

File: source/_docs/automation/modes.markdown [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    trigger:
+      - ...
+    action:
+      - ...
+      - delay: 300  # seconds (=5 minutes)
+```
+
+## Example Queued
+
+Sometimes an automation is doing an action on a device that does not support multiple simultaneous actions. In such cases a queue can be used. In that case the automation will be executed once it's current invocation and queue are done.
```
Comment (zsarnett) [inline source/_docs/automation/modes.markdown]:
```suggestion
Sometimes an automation is doing an action on a device that does not support multiple simultaneous actions. In such cases, a queue can be used. In that case, the automation will be executed once it's current invocation and queue are done.
```

File: source/_docs/automation/services.markdown [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Automation Services"
+description: "How to use the various automation services."
+---
+
```
Comment (frenck) [inline source/_docs/automation/services.markdown:5]: This part of the documentation now feels very out of place.

I think it needs at least a couple of lines to indicate what is on this page.

File: source/_docs/automation/services.markdown [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+### `automation.turn_on`
```
Comment (frenck) [inline source/_docs/automation/services.markdown]:
```suggestion
## Service `automation.turn_on`
```

File: source/_docs/automation/services.markdown [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+This service enables the automation's triggers.
+
+Service data attribute | Optional | Description
+-|-|-
+`entity_id` | no | Entity ID of automation to turn on. Can be a list. `none` or `all` are also accepted.
+
+### `automation.turn_off`
```
Comment (frenck) [inline source/_docs/automation/services.markdown]:
```suggestion
## Service `automation.turn_off`
```

File: source/_docs/automation/services.markdown [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+This service disables the automation's triggers, and optionally stops any currently active actions.
+
+Service data attribute | Optional | Description
+-|-|-
+`entity_id` | no | Entity ID of automation to turn on. Can be a list. `none` or `all` are also accepted.
+`stop_actions` | yes | Stop any currently active actions (defaults to true).
+
+### `automation.toggle`
```
Comment (frenck) [inline source/_docs/automation/services.markdown]:
```suggestion
## Service `automation.toggle`
```

File: source/_docs/automation/services.markdown [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+This service enables the automation's triggers if they were disabled, or disables the automation's triggers, and stops any currently active actions, if the triggers were enabled.
+
+Service data attribute | Optional | Description
+-|-|-
+`entity_id` | no | Entity ID of automation to turn on. Can be a list. `none` or `all` are also accepted.
+
+### `automation.reload`
```
Comment (frenck) [inline source/_docs/automation/services.markdown]:
```suggestion
## Service `automation.reload`
```

File: source/_docs/automation/services.markdown [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+This service reloads all automations, stopping any currently active actions in all of them.
+
+### `automation.trigger`
```
Comment (frenck) [inline source/_docs/automation/services.markdown]:
```suggestion
## Service `automation.trigger`
```

File: source/_docs/automation/using_blueprints.markdown [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+## Blueprint Automations
+
+Automations based on a blueprint only need to be configured to be used. What needs to be configured differs on each blueprint.
+
+To create your first automation based on a blueprint, go to **Configuration** and then **Blueprints**. Find the blueprint that you want to use and click on "Create Automation".
+
+This will open the automation editor with the blueprint selected. Give it a name and configure the blueprint and click on the blue button "Save Automation" in the bottom right.
+
+Done! If you want to revisit the configuration values, you can find it by going to **Configuration** and then **Automations**
```
Comment (frenck) [inline source/_docs/automation/using_blueprints.markdown]:
```suggestion
Done! If you want to revisit the configuration values, you can find it by going to **Configuration** and then **Automations**.
```

File: source/_docs/automation/yaml.markdown [hunk 14]
```diff
@@ -0,0 +1,145 @@
+---
+title: "Automation YAML"
+description: "How to use the automation integration with YAML."
+---
+
+Automations are created in Home Assistant via the UI, but are stored in a YAML format. If you want to edit the YAML of an automation, go to edit the automation, click on the menu button in the top right and turn on YAML mode.
+
+The UI will write your automations to `automations.yaml`. This file is managed by the UI and should not be editted manually.
```
Comment (frenck) [inline source/_docs/automation/yaml.markdown]:
```suggestion
The UI will write your automations to `automations.yaml`. This file is managed by the UI and should not be edited manually.
```

File: source/_docs/automation/yaml.markdown [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+
+It is also possible to write your automations direclty inside `configuration.yaml` or other YAML files. You can do this by adding a labelled `automation` block to your `configuration.yaml`:
+
```
Comment (frenck) [inline source/_docs/automation/yaml.markdown]:
```suggestion
It is also possible to write your automations directly inside `configuration.yaml` or other YAML files. You can do this by adding a labeled `automation` block to your `configuration.yaml`:

```

File: source/_docs/automation/yaml.markdown [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+```yaml
+# The configuration required for the UI to work
+automation: !include automations.yaml
+
+# Labelled automation block
```
Comment (frenck) [inline source/_docs/automation/yaml.markdown]:
```suggestion
# Labeled automation block
```

File: source/_docs/automation/yaml.markdown [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+automation kitchen:
+- trigger:
+    platform: ...
+```
+
+You can add as manny labelled `automation` blocks as you want.
```
Comment (frenck) [inline source/_docs/automation/yaml.markdown]:
```suggestion
You can add as many labeled `automation` blocks as you want.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Nice! 👍

**Pull request merged**